packages/server/config/development.js

```diff
@@ -8,9 +8,10 @@ module.exports = {
         key: process.env.COOKIE_KEY,
         cookie: {
           // options for setCookie, see https://github.com/fastify/fastify-cookie
-          secure: false,
+          secure: true,
```

Comment:
I think we don't need the secure true. With false works the same. In Firefox it's working for me, no in Chrome, even with secure true.